cli/command_policy_set.go

```diff
@@ -95,24 +96,34 @@ func setPolicyFromFlags(ctx context.Context, p *policy.Policy, changeCount *int)
 	return nil
 }
 
-func addRemoveDedupeAndSort(ctx context.Context, desc string, base, add, remove []string, changeCount *int) []string {
+func applyPolicyStringList(ctx context.Context, desc string, val *[]string, add, remove []string, clear bool, changeCount *int) {
```

Comment:
nit / question: what's the rationale to make `val` and in/out parameter instead of returning the new slice as before?

Reply:
it's somewhat easier to copy-paste, nothing else.

Reply:
The reason for the comment is that since a new value is always returned and the original is not really modified, it makes sense to convey that in the function signature.